pkg/docker/build.go

```diff
@@ -74,7 +74,14 @@ func BuildAddLabelsToImage(image string, labels map[string]string) error {
 	args = append(args, ".")
 	cmd := exec.Command("docker", args...)
 
-	dockerfile := "FROM " + image
+	dockerfile := "FROM " + image + "\n"
+	dockerfile += "COPY " + bundledSchemaFile + " .cog\n"
+	const env_path = "/tmp/venv/tools/"
```

Comment:
```suggestion
	env_path := "/tmp/venv/tools/"
```

(Inline const in Go doesn't achieve anything and looks very odd.)